scalableminds/webknossos
# duplicate dataset names

this pr enables us to reference datasets by name + organization and therefore allow duplicate dataset names

### URL of deployed dev instance (used for testing):
- https://duplicatedatasetnames.webknossos.xyz

### Steps to test:
- upload dataset
- create second organization, upload dataset of same name there
- copy URLs, open in private window, should see it if you are allowed to (sharingtoken or public)
- try url without the organization in it (old style) – should be redirected to the oldest dataset you are allowed to see (see below for reasoning behind this heuristic)

### Issues:
- fixes #1878 

------
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
